### DISCUSSION TITLE
Is this repo supports batch request.

### DISCUSSION THREAD

**Discussion by Khushbukela** [shurcooL/graphql#91] [General]
so I can hit one graphql API (endpoint) with multiple queries. 
is this possible

**Comment by dmitshur** [comment]
Are the multiple queries dynamic (computed at runtime) or static (known in advance)? This makes a difference.

It's quite straightforward to use the API of this package to make static batched requests. It's also possible to use reflection to make dynamic batched requests, though it can get somewhat complex. The best place I can link you to for more details is https://github.com/shurcooL/githubv4/issues/17#issuecomment-437646874, and see https://github.com/shurcooL/notifications/commit/926403120bbcbee30699b0ea339c947775dac05f as an example.

There are probably ways to make the experience better, and it's possible others may have better suggestions.
